falco/OWNERS

```diff
@@ -1,3 +1,2 @@
 emeritus_approvers:
   - bencer
-  - nestorsalceda
```

Comment:
this is ok to remove?

Reply:
I replicated the same changes of #373 from @nestorsalceda due to https://github.com/falcosecurity/falco/issues/2143#issuecomment-1195195353.

From https://github.com/falcosecurity/falco/issues/2143, he asks to be removed from everywhere. The linked PRs also  involve `emeritus_approvers` without providing specific rationale for that section.

Reply:
got it, ok 🤔